pkg/actors/actors.go

```diff
@@ -71,7 +71,8 @@ type Options struct {
 	Healthz            healthz.Healthz
```

Comment:
Please add a new `docs/release_notes/v1.15.1.md` for this change.

Reply:
Added